### PR TITLE
Add extra QOL items and stamps to the Listening Post

### PR DESCRIPTION
**Pull request by sheepism1**
## About the PR

Added a blender, a bruise-o-mat instead of an emagged booze-o-mat, binary keys for Listening Post ops, more stamps in the forgery locker, station warrant paper, a blood-red phone, and a kitty!
## Why / Balance

Binary keys to spy on the silicon too, blenders so they can make meds (they could just eat it before) and additional stamps to ensure there's plenty of tomfoolery that you can get up to impersonating the Psychologist, Chief Justice, Lawyer, or Greytide. Balance for the kitty isn't really required cause it's just fluff
## Technical details

Added binary key to interceptor headsets (NOT THE NUKIE OR TRAITOR ONES, ONLY THE LP ONES)
Added a kitty spawner (generic)
Added a bruise-o-mat
Blender for.. I dunno, more options to do in Syndicate Botany. LARP as a synthesis specialist!
Stamps are put in the locker, not prototyped cause fun fact it's just a renamed locker
Station warrant paper for more smoke and mirrors :trollface: 
## Media

QOLed_on_godo
## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

🆑 
- tweak: Added QOL items, additional stamps, and more to the Listening Post, and binary keys are now on 
Listening Post Operative headsets by default. Not even the Borgs are safe from snooping!

### CODE REVIEW

**Comment by sheepism1** [comment]
I forgot a good name, check my next pr instead

**Comment by sheepism1** [comment]
#2004

**Comment by sheepism1** [comment]
we're back :godo:

**Comment by sheepism1** [comment]
passes checks on my repo

**Comment by sheepism1** [comment]
I dunno if I ran  `fixgridatmos` :trollface:

File: Resources/Maps/Shuttles/DeltaV/listening_post.yml
Comment (sheepism1) [inline Resources/Maps/Shuttles/DeltaV/listening_post.yml:1]: map file

**Comment by sheepism1** [comment]
works in game 👍

**Comment by sheepism1** [comment]
cat intentionally kills the mouse-toy when it spawns, but I doubt that'll be an issue tho

**Comment by sheepism1** [comment]
I hope #2009 works w/ this, (it should, its gonna be fun to have the Smoke and Mirrors NEARLY OVERHAUL of the LP)

**Comment by perryprog** [comment]
This might be off limits but... I have had times as LPO where I really wanted a CentComm stamp. That might be asking for too much, though. :)

**Comment by Radezolid** [comment]
> This might be off limits but... I have had times as LPO where I really wanted a CentComm stamp. That might be asking for too much, though. :)

That is a big no, CC stamp is meant to be a certified document from admins, LPOs wouldn't be able to get their hands on it. Best chance you get is CC responding to a fax of yours and forgetting to make it non-editable.

File: Resources/Maps/Shuttles/DeltaV/listening_post.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ entities:
     - type: MetaData
       name: unknown
     - type: Transform
+      parent: invalid
```
Comment (deltanedas) [inline Resources/Maps/Shuttles/DeltaV/listening_post.yml:28]: remove all invalids

**Review by deltanedas — CHANGES_REQUESTED** [review]
the stamps should just be made into a prototype (EntityTablePrototye + a locker that contains it) so adding new stamps doesnt require mapping

**Comment by TadJohnson00** [comment]
@sheepism1 I see that UnknownGhost has taken up this PR in #2155. Do you want this PR closed?

**Comment by sheepism1** [comment]
feel free to close - continuation is approved

**Comment by deltanedas** [comment]
#2155

**Pull request closed without merging**